helix-view/src/theme.rs

```diff
@@ -265,6 +266,14 @@ impl Theme {
         self.highlights[index]
     }
 
+    pub fn name(&self) -> &str {
+        &self.name
+    }
+
+    pub fn with_name(self, name: String) -> Self {
+        Theme { name, ..self }
+    }
```

Comment:
I think this helper is not necessary - I don't think it will be used elsewhere in the future. In the usage above this can be done inline

Reply:
@the-mikedavis - branch updated.